### PR TITLE
docs: github trending tab

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

File: public/sitemap.xml [hunk 1]
```diff
@@ -12,6 +12,9 @@
 <url>
   <loc>https://star-history.com/blog/add-a-live-star-history-chart-to-your-github-readme</loc>
 </url>
+<url>
+  <loc>https://star-history.com/blog/add-a-live-star-history-chart-to-your-github-readme</loc>
+</url>
```
Comment (RainbowDashy) [inline public/sitemap.xml:17]: duplicated?

File: public/blog/github-trending-tab.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+When I learned GitHub’s plan to remove the Trending tab. My immediate reaction was:
+
+![throw.gif](/blog/assets/github-trending-tab/throw.gif)
+
+For open source projects, showing up on the GitHub Trending Tab is a braggable milestone. As someone from the HN suggests, the Trending Tab sets on many developer’s bookmark bar:
+
+![throw.gif](/blog/assets/github-trending-tab/hn.webp)
+
+For such a popular industry leaderboard, it’s always a challenge to be fair and pick the top N repos most people are happy with. And fair is even more subjective in the developer world.
+* Go vs. Rust?
```
Comment (boojack) [inline public/blog/github-trending-tab.md:12]:
```suggestion

* Go vs. Rust?
```

File: public/blog/github-trending-tab.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+While composing this post, I just checked [github.com](https://github.com), and they still have a dedicated Community tab. To me, Community is what GitHub differentiates itself from other VCS providers like GitLab, Bitbucket.
+
+![github-home](/blog/assets/github-trending-tab/github-home.webp)
+
+GitHub calls itself the home for the Developers. The stars, the issues, the discussions, the followers, the sponsors, and the trendings, it’s these engagements make developers have emotional bindings with GitHub.
+
+I wish GitHub could publish a detailed blog to explain the rationale behind this decision. The "low usage" reason doesn’t resonate with the [developer community](https://github.com/community/community/discussions/31643) and even sounds like an April fool’s joke. Maybe GitHub doesn’t realize it, but removing GitHub Trending could significantly weaken the emotional bindings between the developers and the platform.
+
+There do exist 3rd party websites that provide similar or even more powerful trending features, but none can match the built-in user experience and community vibe GitHub Trending Tab provides.
```
Comment (RainbowDashy) [inline public/blog/github-trending-tab.md:43]:
```suggestion
There does exist 3rd party websites that provide similar or even more powerful trending features, but none can match the built-in user experience and community vibe GitHub Trending Tab provides.
```

Comment (tianzhou) [reply public/blog/github-trending-tab.md:43]: This is do

File: public/sitemap.xml (hunk 1 above)
Comment (tianzhou) [reply public/sitemap.xml]: Done

File: public/blog/github-trending-tab.md (hunk 3 above)
Comment (milasuperstar) [reply public/blog/github-trending-tab.md:44]: I'd say "There are" or "3rd party websites do exist"... 
"There do exist 3rd party websites" sounds weird imo

**Review by tianzhou — COMMENTED** [review]
PTAL

File: public/blog/github-trending-tab.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+## News
+
+When I learned GitHub’s plan to remove the Trending tab. My immediate reaction was:
+
+![throw.gif](/blog/assets/github-trending-tab/throw.gif)
+
+For open source projects, showing up on the GitHub Trending Tab is a braggable milestone. As someone from the [HN](https://news.ycombinator.com/item?id=32681319) suggests, the Trending Tab sets on many developer’s bookmark bar:
+
+![throw.gif](/blog/assets/github-trending-tab/hn.webp)
```
Comment (dragonly) [inline public/blog/github-trending-tab.md:9]:
```suggestion
![hn.webp](/blog/assets/github-trending-tab/hn.webp)
```

File: public/blog/github-trending-tab.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+## News
+
+When I learned GitHub’s plan to remove the Trending tab. My immediate reaction was:
+
+![throw.gif](/blog/assets/github-trending-tab/throw.gif)
+
+For open source projects, showing up on the GitHub Trending Tab is a braggable milestone. As someone from the [HN](https://news.ycombinator.com/item?id=32681319) suggests, the Trending Tab sets on many developer’s bookmark bar:
+
+![hn.webp](/blog/assets/github-trending-tab/hn.webp)
```
Comment (dragonly) [inline public/blog/github-trending-tab.md:9]: nit: To be consistent with the contents later, they should be `[throw]` and `[hn]`.

File: public/blog/github-trending-tab.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+![github-home](/blog/assets/github-trending-tab/github-home.webp)
+
+GitHub calls itself the home for the Developers. The stars, the issues, the discussions, the followers, the sponsors, and the trendings, it’s these engagements make developers have emotional bindings with GitHub.
+
+I wish GitHub could publish a detailed blog to explain the rationale behind this decision. The "low usage" reason doesn’t resonate with the [developer community](https://github.com/community/community/discussions/31643) and even sounds like an April fool’s joke. Maybe GitHub doesn’t realize it, but removing GitHub Trending could significantly weaken the emotional bindings between the developers and the platform.
+
+There do exist 3rd party websites that provide similar or even more powerful trending features, but none can match the built-in user experience and community vibe GitHub Trending Tab provides.
+
+Sad to see a step backward in the developer world 😢
```
Comment (dragonly) [inline public/blog/github-trending-tab.md:46]: Do we need a period somewhere (I'm not sure about the emoji) at the end of the sentence?

**Comment by dragonly** [comment]
LG otherwise.

File: public/blog/github-trending-tab.md (hunk 6 above)
Comment (tianzhou) [reply public/blog/github-trending-tab.md:46]: I am not sure as well, just choose one looks better

File: public/blog/github-trending-tab.md (hunk 5 above)
Comment (tianzhou) [reply public/blog/github-trending-tab.md]: Done

File: public/blog/github-trending-tab.md (hunk 6 above)
Comment (dragonly) [reply public/blog/github-trending-tab.md:46]: https://writing.stackexchange.com/questions/51030/should-an-emoji-come-before-or-after-a-full-stop
People are crazy about emoji grammar 🤯

**Review by dragonly — APPROVED** [review]

File: public/blog/github-trending-tab.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+## News
+
+When I learned GitHub’s plan to remove the Trending tab. My immediate reaction was:
+
+![throw](/blog/assets/github-trending-tab/throw.gif)
+
+For open source projects, showing up on the GitHub Trending Tab is a braggable milestone. As someone from the [HN](https://news.ycombinator.com/item?id=32681319) suggests, the Trending Tab sets on many developer’s bookmark bar:
```
Comment (milasuperstar) [inline public/blog/github-trending-tab.md:7]:
```suggestion
For open source projects, showing up on the GitHub Trending Tab is a braggable milestone. As someone from [HN](https://news.ycombinator.com/item?id=32681319) suggests, the Trending Tab sits on many developer’s bookmark bar:
```

File: public/blog/github-trending-tab.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+![hn](/blog/assets/github-trending-tab/hn.webp)
+
+For such a popular industry leaderboard, it’s always a challenge to be fair and pick the top N repos most people are happy with. And fair is even more subjective in the developer world.
```
Comment (milasuperstar) [inline public/blog/github-trending-tab.md:11]:
```suggestion
For such a popular industry leaderboard, it’s always a challenge to be fair and pick the top N projects most people are happy with. And fair is even more subjective in the developer world.
```

**Review by boojack — DISMISSED** [review]
LGTM

File: public/blog/github-trending-tab.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+And apparently, the GitHub folks don’t have enough capacity to clean up the weed. The GitHub Trending algorithm is indeed flawed, but abruptly shutting down the entire service seems like throwing the bath water out with the baby.
+
+## Star History
+
+star-history.com runs a Twitter account [@StartHistoryHQ](https://twitter.com/starhistoryhq) posting interesting GitHub repos every day.
+
+![star-history](/blog/assets/github-trending-tab/star-history.webp)
+
+Some people DMed us asking if we use a bot to find those repos. Actually, we curate each repo manually, and GitHub Trending Tab is our primary source to find those repos. For us, the Trending Tab serves as a starting point. We review the list every day and pick 1 or 2 interesting repos. Without GitHub Trending, finding the needles in the haystack would require much more effort.
```
Comment (milasuperstar) [inline public/blog/github-trending-tab.md:28]: *one of our primary sources hehehehe

**Review by milasuperstar — COMMENTED** [review]
LG otherwise 💚

File: public/blog/github-trending-tab.md (hunk 9 above)
Comment (tianzhou) [reply public/blog/github-trending-tab.md]: Done

**Comment by tianzhou** [comment]
PTAL

**Pull request merged**